Assert-parameters incorrect order

I've seen that many (all?) parameters to the Assert-methods are in the wrong order. For example:
```Assert.assertEquals(7, triggers.size());```

See method definition:
```org.testng.Assert.assertEquals(int actual, int expected)```

This differs from JUnit, where the expected value is usually first, maybe that's where it's been a mixup. Anyway, just thought put an issue for it, since it gives some weird and confusing outputs:
```java.lang.AssertionError: expected [0] but found [7]```

Btw, out of curiousity, is there any particular reason for using TestNG over JUnit? I've never worked with TestNG before.

--- a/src/test/java/com/bikeemotion/quartz/QuartzTest.java
+++ b/src/test/java/com/bikeemotion/quartz/QuartzTest.java
@@ -55,17 +55,17 @@ import org.testng.annotations.AfterClass;
 import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.fail;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertTrue;
-
 import org.quartz.impl.triggers.SimpleTriggerImpl;
 import org.testng.annotations.AfterMethod;
 import org.testng.annotations.BeforeMethod;
 
 import java.util.UUID;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertFalse;
+import static org.testng.Assert.assertTrue;
+import static org.testng.Assert.fail;
+import static org.testng.AssertJUnit.assertNotNull;
 
 @Test(suiteName = "QuartzTest")
 public class QuartzTest extends AbstractTest {
@@ -235,12 +235,12 @@ public class QuartzTest extends AbstractTest {
         .storeDurably()
         .build();
 
-    assertFalse("Unexpected existence of job named 'j1'.", scheduler.checkExists(jobKey("j1")));
+    assertFalse(scheduler.checkExists(jobKey("j1")), "Unexpected existence of job named 'j1'.");
 
     scheduler.addJob(job, false);
 
-    assertTrue("Expected existence of job named 'j1' but checkExists return false.",
-        scheduler.checkExists(jobKey("j1")));
+    assertTrue(scheduler.checkExists(jobKey("j1")),
+        "Expected existence of job named 'j1' but checkExists return false.");
 
     job = scheduler.getJobDetail(jobKey("j1"));
 
@@ -257,14 +257,14 @@ public class QuartzTest extends AbstractTest {
             .withIntervalInSeconds(5))
         .build();
 
-    assertFalse("Unexpected existence of trigger named '11'.", scheduler.checkExists(triggerKey("t1")));
+    assertFalse(scheduler.checkExists(triggerKey("t1")), "Unexpected existence of trigger named '11'.");
 
     scheduler.scheduleJob(job, trigger);
     //give time to hazelcast store the trigger
     Thread.sleep(25);
 
-    assertTrue("Expected existence of trigger named 't1' but checkExists return false.",
-        scheduler.checkExists(triggerKey("t1")));
+    assertTrue(scheduler.checkExists(triggerKey("t1")),
+        "Expected existence of trigger named 't1' but checkExists return false.");
 
     job = scheduler.getJobDetail(jobKey("j1"));
 
@@ -311,23 +311,23 @@ public class QuartzTest extends AbstractTest {
     List<String> jobGroups = scheduler.getJobGroupNames();
     List<String> triggerGroups = scheduler.getTriggerGroupNames();
 
-    assertTrue("Job group list size expected to be = 2 ", jobGroups.size() == 2);
-    assertTrue("Trigger group list size expected to be = 2 ", triggerGroups.size() == 2);
+    assertTrue(jobGroups.size() == 2, "Job group list size expected to be = 2 ");
+    assertTrue(triggerGroups.size() == 2, "Trigger group list size expected to be = 2 ");
 
     Set<JobKey> jobKeys = scheduler.getJobKeys(GroupMatcher.jobGroupEquals(JobKey.DEFAULT_GROUP));
     Set<TriggerKey> triggerKeys = scheduler.getTriggerKeys(GroupMatcher.triggerGroupEquals(TriggerKey.DEFAULT_GROUP));
 
-    assertTrue("Number of jobs expected in default group was 1 ", jobKeys.size() == 1);
-    assertTrue("Number of triggers expected in default group was 1 ", triggerKeys.size() == 1);
+    assertTrue(jobKeys.size() == 1, "Number of jobs expected in default group was 1 ");
+    assertTrue(triggerKeys.size() == 1, "Number of triggers expected in default group was 1 ");
 
     jobKeys = scheduler.getJobKeys(GroupMatcher.jobGroupEquals("g1"));
     triggerKeys = scheduler.getTriggerKeys(GroupMatcher.triggerGroupEquals("g1"));
 
-    assertTrue("Number of jobs expected in 'g1' group was 2 ", jobKeys.size() == 2);
-    assertTrue("Number of triggers expected in 'g1' group was 2 ", triggerKeys.size() == 2);
+    assertTrue(jobKeys.size() == 2, "Number of jobs expected in 'g1' group was 2 ");
+    assertTrue(triggerKeys.size() == 2, "Number of triggers expected in 'g1' group was 2 ");
 
     Trigger.TriggerState s = scheduler.getTriggerState(triggerKey("t2", "g1"));
-    assertTrue("State of trigger t2 expected to be NORMAL ", s.equals(Trigger.TriggerState.NORMAL));
+    assertTrue(s.equals(Trigger.TriggerState.NORMAL), "State of trigger t2 expected to be NORMAL ");
 
     scheduler.pauseTrigger(triggerKey("t2", "g1"));
     s = scheduler.getTriggerState(triggerKey("t2", "g1"));
@@ -335,10 +335,10 @@ public class QuartzTest extends AbstractTest {
 
     scheduler.resumeTrigger(triggerKey("t2", "g1"));
     s = scheduler.getTriggerState(triggerKey("t2", "g1"));
-    assertTrue("State of trigger t2 expected to be NORMAL ", s.equals(Trigger.TriggerState.NORMAL));
+    assertTrue(s.equals(Trigger.TriggerState.NORMAL), "State of trigger t2 expected to be NORMAL ");
 
     Set<String> pausedGroups = scheduler.getPausedTriggerGroups();
-    assertTrue("Size of paused trigger groups list expected to be 0 ", pausedGroups.size() == 0);
+    assertTrue(pausedGroups.isEmpty(), "Size of paused trigger groups list expected to be 0 ");
 
     scheduler.pauseTriggers(GroupMatcher.triggerGroupEquals("g1"));
 
@@ -362,7 +362,7 @@ public class QuartzTest extends AbstractTest {
     Thread.sleep(25);
 
     pausedGroups = scheduler.getPausedTriggerGroups();
-    assertTrue("Size of paused trigger groups list expected to be 1 ", pausedGroups.size() == 1);
+    assertTrue(pausedGroups.size() == 1, "Size of paused trigger groups list expected to be 1 ");
 
     s = scheduler.getTriggerState(triggerKey("t2", "g1"));
     assertEquals(s, Trigger.TriggerState.PAUSED);
@@ -372,32 +372,32 @@ public class QuartzTest extends AbstractTest {
 
     scheduler.resumeTriggers(GroupMatcher.triggerGroupEquals("g1"));
     s = scheduler.getTriggerState(triggerKey("t2", "g1"));
-    assertTrue("State of trigger t2 expected to be NORMAL ", s.equals(Trigger.TriggerState.NORMAL));
+    assertTrue(s.equals(Trigger.TriggerState.NORMAL), "State of trigger t2 expected to be NORMAL ");
     s = scheduler.getTriggerState(triggerKey("t4", "g1"));
-    assertTrue("State of trigger t4 expected to be NORMAL ", s.equals(Trigger.TriggerState.NORMAL));
+    assertTrue(s.equals(Trigger.TriggerState.NORMAL), "State of trigger t4 expected to be NORMAL ");
     pausedGroups = scheduler.getPausedTriggerGroups();
-    assertTrue("Size of paused trigger groups list expected to be 0 ", pausedGroups.size() == 0);
+    assertTrue(pausedGroups.isEmpty(), "Size of paused trigger groups list expected to be 0 ");
 
-    assertFalse("Scheduler should have returned 'false' from attempt to unschedule non-existing trigger. ",
-        scheduler.unscheduleJob(triggerKey("foasldfksajdflk")));
+    assertFalse(scheduler.unscheduleJob(triggerKey("foasldfksajdflk")),
+        "Scheduler should have returned 'false' from attempt to unschedule non-existing trigger. ");
 
-    assertTrue("Scheduler should have returned 'true' from attempt to unschedule existing trigger. ",
-        scheduler.unscheduleJob(triggerKey("t3", "g1")));
+    assertTrue(scheduler.unscheduleJob(triggerKey("t3", "g1")),
+        "Scheduler should have returned 'true' from attempt to unschedule existing trigger. ");
 
     jobKeys = scheduler.getJobKeys(GroupMatcher.jobGroupEquals("g1"));
     triggerKeys = scheduler.getTriggerKeys(GroupMatcher.triggerGroupEquals("g1"));
 
-    assertTrue("Number of jobs expected in 'g1' group was 1 ", jobKeys.size() == 2); // job should have been deleted also, because it is non-durable
-    assertTrue("Number of triggers expected in 'g1' group was 1 ", triggerKeys.size() == 2);
+    assertTrue(jobKeys.size() == 2, "Number of jobs expected in 'g1' group was 1 "); // job should have been deleted also, because it is non-durable
+    assertTrue(triggerKeys.size() == 2, "Number of triggers expected in 'g1' group was 1 ");
 
-    assertTrue("Scheduler should have returned 'true' from attempt to unschedule existing trigger. ",
-        scheduler.unscheduleJob(triggerKey("t1")));
+    assertTrue(scheduler.unscheduleJob(triggerKey("t1")),
+        "Scheduler should have returned 'true' from attempt to unschedule existing trigger. ");
 
     jobKeys = scheduler.getJobKeys(GroupMatcher.jobGroupEquals(JobKey.DEFAULT_GROUP));
     triggerKeys = scheduler.getTriggerKeys(GroupMatcher.triggerGroupEquals(TriggerKey.DEFAULT_GROUP));
 
-    assertTrue("Number of jobs expected in default group was 1 ", jobKeys.size() == 1); // job should have been left in place, because it is non-durable
-    assertTrue("Number of triggers expected in default group was 0 ", triggerKeys.size() == 0);
+    assertTrue(jobKeys.size() == 1, "Number of jobs expected in default group was 1 "); // job should have been left in place, because it is non-durable
+    assertTrue(triggerKeys.isEmpty(), "Number of triggers expected in default group was 0 ");
 
   }
 
@@ -406,18 +406,17 @@ public class QuartzTest extends AbstractTest {
     throws Exception {
 
     // test basic storage functions of scheduler...
-
     JobDetail job = newJob()
         .ofType(MyJob.class)
         .withIdentity("j1")
         .storeDurably()
         .build();
 
-    assertFalse("Unexpected existence of job named 'j1'.", scheduler.checkExists(jobKey("j1")));
+    assertFalse(scheduler.checkExists(jobKey("j1")), "Unexpected existence of job named 'j1'.");
 
     scheduler.addJob(job, false);
 
-    assertTrue("Unexpected non-existence of job named 'j1'.", scheduler.checkExists(jobKey("j1")));
+    assertTrue(scheduler.checkExists(jobKey("j1")), "Unexpected non-existence of job named 'j1'.");
 
     JobDetail nonDurableJob = newJob()
         .ofType(MyJob.class)
@@ -428,16 +427,16 @@ public class QuartzTest extends AbstractTest {
       scheduler.addJob(nonDurableJob, false);
       fail("Storage of non-durable job should not have succeeded.");
     } catch (SchedulerException expected) {
-      assertFalse("Unexpected existence of job named 'j2'.", scheduler.checkExists(jobKey("j2")));
+      assertFalse(scheduler.checkExists(jobKey("j2")), "Unexpected existence of job named 'j2'.");
     }
 
     scheduler.addJob(nonDurableJob, false, true);
 
-    assertTrue("Unexpected non-existence of job named 'j2'.", scheduler.checkExists(jobKey("j2")));
+    assertTrue(scheduler.checkExists(jobKey("j2")), "Unexpected non-existence of job named 'j2'.");
   }
 
   //TODO: FIX IT (URGENT)
-  @Test(enabled = false)
+  @Test
   public void testShutdownWithSleepReturnsAfterAllThreadsAreStopped()
     throws Exception {
 
@@ -452,7 +451,7 @@ public class QuartzTest extends AbstractTest {
     Thread.sleep(1500L);
 
     Map<Thread, StackTraceElement[]> allThreadsEnd = Thread.getAllStackTraces();
-    Set<Thread> endingThreads = new HashSet<Thread>(allThreadsEnd.keySet());
+    Set<Thread> endingThreads = new HashSet<>(allThreadsEnd.keySet());
     // remove all pre-existing threads from the set
     for (Thread t : allThreadsStart.keySet()) {
       allThreadsEnd.remove(t);
@@ -490,7 +489,7 @@ public class QuartzTest extends AbstractTest {
     }
     Thread.sleep(2000L);
 
-    assertTrue("Found unexpected new threads (see console output for listing)", allThreadsEnd.size() == 0);
+    assertTrue(allThreadsEnd.isEmpty(), "Found unexpected new threads (see console output for listing)");
   }
 
   @Test
@@ -517,7 +516,7 @@ public class QuartzTest extends AbstractTest {
 
     long fTime = jobExecTimestamps.get(0);
 
-    assertTrue("Immediate trigger did not fire within a reasonable amount of time.", (fTime - sTime < 7000L)); // This is dangerously subjective!  but what else to do?
+    assertTrue((fTime - sTime < 7000L), "Immediate trigger did not fire within a reasonable amount of time."); // This is dangerously subjective!  but what else to do?
   }
 
   @Test
@@ -545,7 +544,7 @@ public class QuartzTest extends AbstractTest {
 
     long fTime = jobExecTimestamps.get(0);
 
-    assertTrue("Immediate trigger did not fire within a reasonable amount of time.", (fTime - sTime < 7000L)); // This is dangerously subjective!  but what else to do?
+    assertTrue((fTime - sTime < 7000L), "Immediate trigger did not fire within a reasonable amount of time."); // This is dangerously subjective!  but what else to do?
   }
 
   @Test
@@ -570,7 +569,7 @@ public class QuartzTest extends AbstractTest {
 
     long fTime = jobExecTimestamps.get(0);
 
-    assertTrue("Immediate trigger did not fire within a reasonable amount of time.", (fTime - sTime < 7000L)); // This is dangerously subjective!  but what else to do?
+    assertTrue((fTime - sTime < 7000L), "Immediate trigger did not fire within a reasonable amount of time."); // This is dangerously subjective!  but what else to do?
   }
 
   @Test
@@ -601,7 +600,7 @@ public class QuartzTest extends AbstractTest {
     Thread.sleep(100);
 
     List<? extends Trigger> triggersOfJob = scheduler.getTriggersOfJob(job.getKey());
-    assertEquals(2, triggersOfJob.size());
+    assertEquals(triggersOfJob.size(), 2);
     assertTrue(triggersOfJob.contains(trigger1));
     assertTrue(triggersOfJob.contains(trigger2));
 
@@ -689,6 +688,7 @@ public class QuartzTest extends AbstractTest {
 
   public static class TestJobWithSync implements Job {
 
+    @Override
     public void execute(JobExecutionContext context)
       throws JobExecutionException {
 
@@ -711,6 +711,7 @@ public class QuartzTest extends AbstractTest {
   @PersistJobDataAfterExecution
   public static class TestAnnotatedJob implements Job {
 
+    @Override
     public void execute(JobExecutionContext context)
       throws JobExecutionException {
 

--- a/src/test/java/com/bikeemotion/quartz/jobstore/hazelcast/HazelcastJobStoreTest.java
+++ b/src/test/java/com/bikeemotion/quartz/jobstore/hazelcast/HazelcastJobStoreTest.java
@@ -251,14 +251,14 @@ public class HazelcastJobStoreTest extends AbstractTest {
     jobStore.storeTrigger(trigger5, false);
 
     List<OperableTrigger> acquiredTriggers = jobStore.acquireNextTriggers(baseFireTime + 1500, 3, 1000L);
-    assertEquals(3, acquiredTriggers.size());
+    assertEquals(acquiredTriggers.size(), 3);
 
     jobStore.releaseAcquiredTrigger(trigger1);
     jobStore.releaseAcquiredTrigger(trigger2);
     jobStore.releaseAcquiredTrigger(trigger3);
 
     acquiredTriggers = jobStore.acquireNextTriggers(baseFireTime + 1500, 4, 1000L);
-    assertEquals(4, acquiredTriggers.size());
+    assertEquals(acquiredTriggers.size(), 4);
 
     jobStore.releaseAcquiredTrigger(trigger1);
     jobStore.releaseAcquiredTrigger(trigger2);
@@ -266,18 +266,18 @@ public class HazelcastJobStoreTest extends AbstractTest {
     jobStore.releaseAcquiredTrigger(trigger4);
 
     acquiredTriggers = jobStore.acquireNextTriggers(baseFireTime + 1500, 5, 1000L);
-    assertEquals(4, acquiredTriggers.size());
+    assertEquals(acquiredTriggers.size(), 4);
 
     jobStore.releaseAcquiredTrigger(trigger1);
     jobStore.releaseAcquiredTrigger(trigger2);
 
-    assertEquals(1, jobStore.acquireNextTriggers(baseFireTime + 2001, 5, 0L).size());
+    assertEquals(jobStore.acquireNextTriggers(baseFireTime + 2001, 5, 0L).size(), 1);
     jobStore.releaseAcquiredTrigger(trigger1);
 
-    assertEquals(2, jobStore.acquireNextTriggers(baseFireTime + 2000, 5, 150L).size());
+    assertEquals(jobStore.acquireNextTriggers(baseFireTime + 2000, 5, 150L).size(), 2);
     jobStore.releaseAcquiredTrigger(trigger1);
 
-    assertEquals(1, jobStore.acquireNextTriggers(baseFireTime + 2000, 5, 150L).size());
+    assertEquals(jobStore.acquireNextTriggers(baseFireTime + 2000, 5, 150L).size(), 1);
     jobStore.releaseAcquiredTrigger(trigger1);
 
     jobStore.removeTrigger(trigger1.getKey());
@@ -302,16 +302,16 @@ public class HazelcastJobStoreTest extends AbstractTest {
 
     trigger.computeFirstFireTime(null);
 
-    assertEquals(Trigger.TriggerState.NONE, jobStore.getTriggerState(trigger.getKey()));
+    assertEquals(jobStore.getTriggerState(trigger.getKey()), Trigger.TriggerState.NONE);
 
     jobStore.storeTrigger(trigger, false);
-    assertEquals(Trigger.TriggerState.NORMAL, jobStore.getTriggerState(trigger.getKey()));
+    assertEquals(jobStore.getTriggerState(trigger.getKey()), Trigger.TriggerState.NORMAL);
 
     jobStore.pauseTrigger(trigger.getKey());
-    assertEquals(Trigger.TriggerState.PAUSED, jobStore.getTriggerState(trigger.getKey()));
+    assertEquals(jobStore.getTriggerState(trigger.getKey()), Trigger.TriggerState.PAUSED);
 
     jobStore.resumeTrigger(trigger.getKey());
-    assertEquals(Trigger.TriggerState.NORMAL, jobStore.getTriggerState(trigger.getKey()));
+    assertEquals(jobStore.getTriggerState(trigger.getKey()), Trigger.TriggerState.NORMAL);
 
     OperableTrigger rt1 = jobStore.acquireNextTriggers(
         new Date(trigger.getNextFireTime().getTime()).getTime() + 10000,
@@ -328,7 +328,7 @@ public class HazelcastJobStoreTest extends AbstractTest {
         .get(0);
 
     assertNotNull(rt2);
-    assertEquals(rt1.getJobKey(),rt2.getJobKey());
+    assertEquals(rt2.getJobKey(), rt1.getJobKey());
 
     assertTrue(jobStore.acquireNextTriggers(new Date(rt2.getNextFireTime().getTime()).getTime() + 1500,
         1,
@@ -347,7 +347,7 @@ public class HazelcastJobStoreTest extends AbstractTest {
     tr.setCalendarName(null);
 
     jobStore.storeTrigger(tr, false);
-    assertEquals(tr, jobStore.retrieveTrigger(tr.getKey()));
+    assertEquals(jobStore.retrieveTrigger(tr.getKey()), tr);
 
     try {
       jobStore.storeTrigger(tr, false);
@@ -358,8 +358,8 @@ public class HazelcastJobStoreTest extends AbstractTest {
 
     tr.setCalendarName("QQ");
     jobStore.storeTrigger(tr, true);
-    assertEquals(tr, jobStore.retrieveTrigger(tr.getKey()));
-    assertEquals("QQ", jobStore.retrieveTrigger(tr.getKey()).getCalendarName(), "StoreJob doesn't replace triggers");
+    assertEquals(jobStore.retrieveTrigger(tr.getKey()), tr);
+    assertEquals(jobStore.retrieveTrigger(tr.getKey()).getCalendarName(),"QQ", "StoreJob doesn't replace triggers");
   }
 
   @Test()
@@ -382,7 +382,7 @@ public class HazelcastJobStoreTest extends AbstractTest {
         new Date().getTime());
 
     jobStore.storeTrigger(tr, false);
-    assertEquals(Trigger.TriggerState.PAUSED, jobStore.getTriggerState(tr.getKey()));
+    assertEquals(jobStore.getTriggerState(tr.getKey()), Trigger.TriggerState.PAUSED);
   }
 
   @Test()
@@ -408,7 +408,7 @@ public class HazelcastJobStoreTest extends AbstractTest {
     for (int i = 0; i < nJobs; i++) {
       JobKey jobKey = JobKey.jobKey("job" + i);
       JobDetail storedJob = store.retrieveJob(jobKey);
-      Assert.assertEquals(jobKey, storedJob.getKey());
+      Assert.assertEquals(storedJob.getKey(), jobKey);
     }
   }
 
@@ -437,11 +437,11 @@ public class HazelcastJobStoreTest extends AbstractTest {
     for (int i = 0; i < nJobs; i++) {
       JobKey jobKey = JobKey.jobKey("job" + i);
       JobDetail storedJob = store.retrieveJob(jobKey);
-      Assert.assertEquals(jobKey, storedJob.getKey());
+      Assert.assertEquals(storedJob.getKey(), jobKey);
 
       TriggerKey triggerKey = TriggerKey.triggerKey("job" + i);
       OperableTrigger storedTrigger = store.retrieveTrigger(triggerKey);
-      Assert.assertEquals(triggerKey, storedTrigger.getKey());
+      Assert.assertEquals(storedTrigger.getKey(), triggerKey);
     }
   }
 
@@ -477,7 +477,7 @@ public class HazelcastJobStoreTest extends AbstractTest {
       // do. Otherwise
       // the store.acquireNextTriggers() will not work properly.
       Date fireTime = trigger.computeFirstFireTime(null);
-      Assert.assertEquals(true, fireTime != null);
+      Assert.assertNotNull(fireTime);
 
       store.storeJobAndTrigger(job, trigger);
     }
@@ -488,8 +488,8 @@ public class HazelcastJobStoreTest extends AbstractTest {
       int maxCount = 1;
       long timeWindow = 0;
       List<OperableTrigger> triggers = store.acquireNextTriggers(noLaterThan, maxCount, timeWindow);
-      Assert.assertEquals(1, triggers.size());
-      Assert.assertEquals("job" + i, triggers.get(0).getKey().getName());
+      Assert.assertEquals(triggers.size(), 1);
+      Assert.assertEquals(triggers.get(0).getKey().getName(), "job" + i);
 
       // Let's remove the trigger now.
       store.removeJob(triggers.get(0).getJobKey());
@@ -526,7 +526,7 @@ public class HazelcastJobStoreTest extends AbstractTest {
       // do. Otherwise
       // the store.acquireNextTriggers() will not work properly.
       Date fireTime = trigger.computeFirstFireTime(null);
-      Assert.assertEquals(true, fireTime != null);
+      Assert.assertNotNull(fireTime);
 
       store.storeJobAndTrigger(job, trigger);
     }
@@ -538,7 +538,7 @@ public class HazelcastJobStoreTest extends AbstractTest {
     // they are a minute apart
     long timeWindow = 8 * MIN;
     List<OperableTrigger> triggers = store.acquireNextTriggers(noLaterThan, maxCount, timeWindow);
-    Assert.assertEquals(7, triggers.size());
+    Assert.assertEquals(triggers.size(), 7);
   }
 
   @Test
@@ -1182,9 +1182,6 @@ public class HazelcastJobStoreTest extends AbstractTest {
           + "]Should be PAUSED for trigger : [" + trigger.getKey()
           + "] and job [" + trigger.getJobKey() + "]");
       assertEquals(triggerState, Trigger.TriggerState.PAUSED);
-      //          .overridingErrorMessage(
-      //              "Should be PAUSED for trigger : [" + trigger.getKey()
-      //                  + "] and job [" + trigger.getJobKey() + "]");
     }
 
     triggersForJob = jobStore.getTriggersForJob(job3.getKey());
@@ -1333,7 +1330,7 @@ public class HazelcastJobStoreTest extends AbstractTest {
     long firstFireTime = new Date(trigger1.getNextFireTime().getTime()).getTime();
 
     List<OperableTrigger> acquiredTriggers = jobStore.acquireNextTriggers(firstFireTime + 500, 1, 0L);
-    assertEquals(1, acquiredTriggers.size());
+    assertEquals(acquiredTriggers.size(), 1);
 
     List<TriggerFiredResult> triggerFired = jobStore.triggersFired(acquiredTriggers);
     assertEquals(triggerFired.size(), 1);

